apps/mobile/app/import-signers.tsx

```diff
@@ -0,0 +1,66 @@
+import { SectionTitle } from '@/src/components/Title'
+import React from 'react'
+import { ScrollView } from 'react-native-gesture-handler'
+import Seed from '@/assets/images/seed.png'
+import { SafeFontIcon } from '@/src/components/SafeFontIcon'
+import { useScrollableHeader } from '@/src/navigation/useScrollableHeader'
+import { NavBarTitle } from '@/src/components/Title'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { SafeCard } from '@/src/components/SafeCard'
+
+/**
+ * TODO: Add the images for each items
+ * waiting for the design to exprot - figma is crazy
+ */
```

Comment:
does this comment still apply @clovisdasilvaneto

Reply:
Yes, but I removed it in the import signers PR